build-tools/automation/azure-pipelines.yaml

```diff
@@ -159,6 +177,12 @@ stages:
     steps:
     - checkout: self
       submodules: recursive
+    ### PLEASE REMOVE THIS BLOCK IF IN THE DIFF
+    - script: |
+        git checkout master
+        git pull --rebase origin macos-mojave-builds
+        git submodule update --init --recursive
+    ### PLEASE REMOVE THIS BLOCK IF IN THE DIFF
```

Comment:
I think all the custom rebasing logic in the YAML file can be removed now? This should happen automatically when building the PR.